src/organisms/Breadcrumbs.jsx

```diff
@@ -63,22 +63,41 @@ const Breadcrumbs = () => {
     }
   } else if (pathArray[0] === "project-setup") {
     appendToParentList("project setup", "/project-setup");
-    if (pathArray.length === 3) {
-      appendToParentList("Confirmation", "/project-setup/confirmation");
+    if (pathArray[1] !== undefined && pathArray[2] !== undefined) {
+      appendToParentList(pathArray[1], `/project-setup/${pathArray[1]}`);
+    }
+    //Adding this one since HCD is working on adding an extra page after selecting components
+    if (pathArray[2] !== undefined && pathArray[3] !== undefined) {
+      appendToParentList(
+        pathArray[2],
+        `/project-setup/${pathArray[1]}/${pathArray[2]}`
+      );
     }
   } else if (pathArray[0] === "projects") {
```

Comment:
I'd much rather see this code broken into testable units with self-documenting names.  We've deprioritized cypress testing due to the effort of getting it into our pipelines.  Unit testing is much easier to add therefore they are more valuable to us at this time.

Reply:
Cypress test are easy to run and can do proper integration test in under 3 minutes as pointed out above. They add more value then unit test themselves add, that is why they are important.